src/drt/README.md

```diff
@@ -17,15 +17,173 @@ guide format.
+| `-repair_pdn_vias` | this option is used for some PDKs which have M1 and M2 layers that run in parallel. |
+
+### Detailed Route debugging
+
+```tcl
+detailed_route_debug 
+    [-pa]
+    [-ta]
+    [-dr]
+    [-maze]
+    [-net name]
+    [-pin name]
+    [-worker x y]
+    [-iter iter]
+    [-pa_markers]
+    [-dump_dr]
+    [-dump_dir dir]
+    [-pa_edge]
+    [-pa_commit]
+    [-write_net_tracks]
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-pa` | enable debug for pin access |
+| `-ta` | enable debug for track assignment |
+| `-dr` | enable debug for detailed routing |
+| `-maze` | enable debug for maze routing | 
+| `-net` | enable debug for net name |
+| `-pin` | enable debug for pin name |
+| `-worker` | debugs routes that pass through the point `{x, y}` |
+| `-iter` | debug iterations |
+| `-pa_markers` | enable pin access markers |
+| `-dump_dr` | dump detailed routing filename |
+| `-dump_dir` | dump detailed routing directory |
+| `-pa_edge` | visibility of pin access edges |
+| `-pa_commit` | visibility of pin access commits |
+| `-write_net_tracks` | enable writing of net track assigments |
+
+### Check Pin access 
+
+```tcl
+pin_access
+    [-db_process_node name]
+    [-bottom_routing_layer layer]
+    [-top_routing_layer layer]
+    [-min_access_points count]
+    [-verbose level]
+    [-distributed]
+    [-remote_host rhost]
+    [-remote_port rport]
+    [-shared_volume vol]
+    [-cloud_size sz]
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-db_process_node` | specify for process node | 
+| `-bottom_routing_layer` | bottommost routing layer |
+| `-top_routing_layer` | topmost routing layer |
+| `-min_access_points` | minimum number of access points per pin |
+| `-verbose` | set verbose if the value is greater than 0 |
+| `-distributed` | enable distributed pin access algorithm |
+| `-remote_host` | the host IP |
+| `-remote_port` | the value of the port to access from |
+| `-shared_volume` | the mount path of the nfs shared folder |
+| `-cloud_size` | the number of workers |
+
+### Useful developer functions
+
+If you are a developer, you might find these useful. More details can be found in the [source file](./src/TritonRoute.cpp) or the [swig file](./src/TritonRoute.i).
+
+```
+detailed_route_set_default_via [-via]   # set default via 
+detailed_route_set_unidirectional_layer [-layer] # set unidirectional layer
+step_dr # refer to function detailed_route_step_drt
+check_drc   # refer to function check_drc_cmd 
+```
 
 ## Example scripts
 
+Example script demonstrating how to run TritonRoute on a sample design of `gcd`
+in the Nangate45 technology node.
+
+```shell
+./test/gcd_nangate45.tcl    # single machine example
+./test/gcd_nangate45_distributed.tcl    # distributed example
+```
+
 ## Regression tests
 
+There are a set of regression tests in `/test`. For more information, refer to this [section](../../README.md#regression-tests). 
```

Comment:
The link here should point to the correct header under Regressions: https://openroad--3631.org.readthedocs.build/en/3631/main/README.html#regression-tests

Reply:
This seems to be a bug with the theme. I filed an issue with the repo maintainers, so we'll put a pin in this for now